frontend/chat-plugin/src/App.tsx

```diff
@@ -2,16 +2,19 @@ import React, {Component} from 'react';
 
 import style from './App.module.scss';
 import Chat from './components/chat';
+import {getResumeTokenFromStorage} from './storage';
 
 export default class App extends Component {
   render() {
     const queryParams = new URLSearchParams(window.location.search);
     const channelId = queryParams.get('channel_id');
+    const resumeToken = getResumeTokenFromStorage(channelId);
 
     return (
       <div className={style.container}>
         {channelId ? (
           <Chat
+            resumeToken={resumeToken}
```

Comment:
I am not sure this will work with the script version. I think it's better if you get the resumeToken inside the Chat component